utils/cliutils/commandsflags.go

```diff
@@ -0,0 +1,1223 @@
+	},
+	vars: cli.StringFlag{
+		Name:  vars,
+		Usage: "[Optional] List of variables in the form of \"key1=value1;key2=value2;...\" to be replaced in the template. In the template, the variables should be used as follows: ${key1}.` `",
+	},
+	groups: cli.StringFlag{
+		Name: groups,
+		Usage: "[Default: *] A list of comma-separated groups for the access token to be associated with. " +
+			"Specify * to indicate that this is a 'user-scoped token', i.e., the token provides the same access privileges that the current subject has, and is therefore evaluated dynamically. " +
+			"A non-admin user can only provide a scope that is a subset of the groups to which he belongs` `",
+	},
+	grantAdmin: cli.BoolFlag{
+		Name:  grantAdmin,
+		Usage: "[Default: false] Set to true to provides admin privileges to the access token. This is only available for administrators.` `",
+	},
+	expiry: cli.StringFlag{
+		Name:  expiry,
+		Usage: "[Default: " + strconv.Itoa(TokenExpiry) + "] The time in seconds for which the token will be valid. To specify a token that never expires, set to zero. Non-admin can only set a value that is equal to or less than the default 3600.` `",
+	},
+	refreshable: cli.BoolFlag{
+		Name:  refreshable,
+		Usage: "[Default: false] Set to true if you'd like the the token to be refreshable. A refresh token will also be returned in order to be used to generate a new token once it expires.` `",
+	},
+	audience: cli.StringFlag{
+		Name:  audience,
+		Usage: "[Optional] A space-separate list of the other Artifactory instances or services that should accept this token identified by their Artifactory Service IDs, as obtained by the 'jfrog rt curl api/system/service_id' command.` `",
+	},
+	// Xray's commands Flags
+	licenseId: cli.StringFlag{
+		Name:  licenseId,
+		Usage: "[Mandatory] Xray license ID` `",
+	},
+	from: cli.StringFlag{
+		Name:  from,
+		Usage: "[Optional] From update date in YYYY-MM-DD format.` `",
+	},
+	to: cli.StringFlag{
+		Name:  to,
+		Usage: "[Optional] To update date in YYYY-MM-DD format.` `",
+	},
+	version: cli.StringFlag{
+		Name:  version,
+		Usage: "[Optional] Xray API version.` `",
+	},
+	target: cli.StringFlag{
+		Name:  target,
+		Usage: "[Default: ./] Path for downloaded update files.` `",
+	},
+	// Mission Control's commands Flags
+	mcUrl: cli.StringFlag{
+		Name:  url,
+		Usage: "[Optional] Mission Control URL.` `",
+	},
+	mcAccessToken: cli.StringFlag{
+		Name:  accessToken,
+		Usage: "[Optional] Mission Control Admin token.` `",
+	},
+	mcInteractive: cli.BoolTFlag{
+		Name:  interactive,
+		Usage: "[Default: true] Set to false if you do not want the config command to be interactive. If true, the other command options become optional.",
+	},
+	licenseCount: cli.StringFlag{
+		Name:  licenseCount,
+		Value: "",
+		Usage: "[Default: " + strconv.Itoa(DefaultLicenseCount) + "] The number of licenses to deploy. Minimum value is 1.` `",
+	},
+}
+
+var commandFlags = map[string][]string{
+	"config": {
```

Comment:
I suggest using consts for all command keys (for example, "config") and then use these consts from cli.go.
This eliminates the chances for a developer typo.